modules/client-configuration/pages/client-upgrades-major.adoc

```diff
@@ -10,63 +10,61 @@ The upgrade is controlled by {yast} and {ay}, it does not use Zypper.
 
 == Prepare to Migrate
 
-Before you can migrate your client from SLE{nbsp}12 to SLE{nbsp}15{nbsp}, you need to:
+Before you can migrate your client from SLE{nbsp}12 to SLE{nbsp}15, you need to:
 
-.Procedure: Uploading {ay} Profile
 . Prepare installation media
-. Create an auto-installable distribution
+. Declare an auto-installable distribution
 . Create an activation key
-. Upload an {ay} profile
+. Create an autoinstallation profile
```

Comment:
I wonder why is one **auto-installable** distribution and another **autoinstallation** profile.

If it's because that is how we have it n the webui, we have **Autoinstallable** distribution there.

Personally I would use Autoinstallation distribution and profile everywhere.

Reply:
Thanks!  Yes, I also prefer Autoinstallation (with or without dash, I must check it).